File: DeckBuilderElements/postBuilders.js
export class DefaultPostBuilder {
    execute(cards){}
}

export class DeckShuffler{
    execute(cards){
        var shuffled = []
        while(cards.length > 0){
            var index = Math.floor(Math.random() * cards.length)
            shuffled.push(cards[index])
            cards.splice(index, 1)
        }
        return shuffled          
    }
}

File: python/DeckBuilderElements/postBuilders.py
import config
import random

class DefaultPostBuilder:
    def execute(self, cards):
        return cards

class DeckShuffler:
    def execute(self, cards):
        shuffled = []
        while len(cards) > 0:
            index = random.randint(0, len(cards) - 1)
            shuffled.append(cards[index])
            cards.pop(index)
        return shuffled

